src/fetchers/wakatime-embed-fetcher.js

```diff
@@ -0,0 +1,22 @@
+const axios = require("axios");
+
+const wakaTimeEmbedData = async ({ username , embedCode}) => {
+    try {
+        const { data } = await axios.get(
+            `https://wakatime.com/share/@${username}/${embedCode}.json`,
+        );
+
+        return data.data;
+    } catch (err) {
+        if (err.response.status === 404) {
+            throw new Error(
+                "Wakatime Data not found",
+            );
+        }
+        throw err;
+    }
+};
```

Comment:
```suggestion
const wakaTimeEmbedData = ({ username, embedCode }) =>
  axios
    .get(`https://wakatime.com/share/@${username}/${embedCode}.json`)
    .then(({ data: { data } }) => data)
    .catch(err => {
      throw err.response?.status === 404
        ? new Error('Wakatime Data not found')
        : err
    })
```

I think we can save some memory with this code. : )